content/v2/domains/templates.markdown

```diff
@@ -0,0 +1,37 @@
+---
+title: Templates API | Domains | DNSimple API v2 (Beta)
+excerpt: This page documents the DNSimple templates API v2.
```

Comment:
@jacegu I believe this is a common description in all pages. If we make any change, it should be in all pages.

Also, it should look like a welcoming description and end with a dot. It is likely the description that search engines will display.

I don't think it's reasonable to make any change in this PR.